### ISSUE TITLE
license?

### ISSUE THREAD

**Issue by ddiakopoulos** [magnusjonsson/microtracker#4]
Hi @magnusjonsson !

I had carved out your Moog ladder implementation for comparison with a host of implementations others here (https://github.com/ddiakopoulos/MoogLadders/blob/master/src/MicrotrackerModel.h). Right now it's concretely listed as being your copyright, but I'm wondering if you would provide a more explicit license. Thank you!

**Comment by magnusjonsson** [comment]
Thanks, I've added an explicit UNLICENSE file. I'll follow your project with interest!

**Comment by magnusjonsson** [comment]
Hi Dimitri,

That is a really cool project. What will be the ultimate outcome of it you
think? I read README.md but couldn't really see where it was going.

Anyway, I've added an UNLICENSE file, inspired by your choice.

Thanks!

**Comment by ddiakopoulos** [comment]
Heya @magnusjonsson -- thanks for the quick reply and the permissive license! 

The project doesn't really have an explicit goal -- it's just a bit of digital archeology. I implemented a version of the filter for a DSP project in school back in ~2011. 

I kept finding other versions to compare it with and found it might be useful to other people who are interested in virtual analog modeling. I also just find it amusing that so many academics have investigated new techniques into modeling this particular filter.
